ansible/roles/controller/tasks/deploy.yml

```diff
@@ -292,6 +292,19 @@
     env: "{{ env | combine(elastic_env) }}"
   when: db.activation_store.backend == "ElasticSearch"
 
+- name: setup mongodb artifact store env
+  set_fact:
+    mongodb_env:
+      "CONFIG_whisk_mongodb_uri": "{{ db.mongodb.connect_string }}"
+      "CONFIG_whisk_mongodb_database": "{{ db.mongodb.database }}"
+      "CONFIG_whisk_spi_ArtifactStoreProvider": "org.apache.openwhisk.core.database.mongodb.MongoDBArtifactStoreProvider"
+  when: db.artifact_store.backend == "MongoDB"
+
+- name: merge mongodb artifact store env
+  set_fact:
+    env: "{{ env | combine(mongodb_env) }}"
+  when: db.artifact_store.backend == "MongoDB"
```

Comment:
If we do this can we use MongoDB as an activation store and CouchDB as an artifact store?

Reply:
nope, this is for `artifact`, which means all kind of entities

Reply:
What users can do if they want to use the CouchDB artifact store with the MongoDB activation store?

Reply:
I havn't test such case yet

Reply:
ok I think that can be handled in a different PR.